server/routes/trees/trees-router.js

```diff
@@ -38,8 +38,13 @@ treesRouter.post('/', async (req, res) => {
     throw new AppError(400, 'Post Tree: Missing required parameter(s): common, scientific, city, lat, lng.');
   }
   const dateplanted = req.body.planted;
-  const { common, scientific, sourceId,  city, country, lat, lng, address, 
-    height, dbh, owner, health, waterFreq, who, email,
+  const { common, scientific, genus, 
+    lat, lng, sourceId,
+    address, city, country, neighborhood,
+    height, dbh, health, 
+    who, email,owner, volunteer,
+    notes, waterFreq, irrigation,
+    download: url,
```

Comment:
If fields need to be renamed, seems like we should really do it on ingest so they're all consistent.  Doing it here means edited trees will have different shapes than the majority of them.

Also, if the data *is* sent with the correct fields, those will get lost here, if I'm reading it correctly.